camelot/types/__init__.py

```diff
@@ -300,11 +300,10 @@ class Movie( Entity ):
     """
     
     impl = types.Unicode
-    stored_file_implementation = StoredFile
-    
-    def __init__(self, max_length=100, upload_to=u'', storage=Storage, **kwargs):
+
+    def __init__(self, storage=None, *, max_length=100, **kwargs):
```

Comment:
waarom hier dat '*' argument nog ?

maak hier ook het storage argument maar verplicht ... op bepaalde plaatsen, zoals in migraties kan je het dan alsnog expliciet op None zetten, maar eigenlijk moet dat gewoon overal verplicht w ingegeven

Reply:
het * argument zorgt ervoor dat max_length enkel als kwarg kan worden meegegeven, ik had dat gedaan omdat ik de argumenten van plaats had gewisseld dus voor de veiligheid indien max_length niet als kwarg word meegegeven

Reply:
wat je dan beter kan doen is : 

- geef storage geen default
- assert dat storage een instance is van Storage
- max lenght met een default als 2e argument

zo ben je zeker dat de boel er uit knalt als per ongeluk er geen storage zou zijn meegegeven en de custom max length zou w opgepikt als storage ...